src/index.js

```diff
@@ -1,27 +1,110 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
- [ ] Please create a new javascript file for the new functionalities. It's a project requirement and a way of distributing your code for improved readability.

Reply:
Good job following the advice of the previous reviewer